make.py

```diff
@@ -9,16 +9,16 @@
 from postgis import LineString, MultiLineString
 from postgis.asyncpg import register
 
-OVERPASS = 'http://overpass-api.de/api/interpreter'
-
+OVERPASS = 'https://overpass-api.de/api/interpreter'
 
 async def get_relation(conn, **tags):
     if 'iso' in tags:
         tags['ISO3166-1:alpha2'] = tags.pop('iso')
     tags = "".join(f'["{k}"="{v}"]' for k, v in tags.items())
     path = Path('tmp/boundary')
     path.mkdir(parents=True, exist_ok=True)
-    path = path / tags.replace('/', '_')
+    file = tags.replace('/', '_').replace('][', '_').replace('"','').replace(':','_').replace('[', '').replace(']', '')+'.json'
```

Comment:
`file` is a python keyword, so the convention is to name it `file_` not to override it.

Reply:
Oh sorry I forgot it, I changed it to `file_`